velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -140,41 +147,34 @@ void verifyFailures(hdfsFS hdfs) {
           .str();
   auto serverAddress = (boost::format("%s:%s") % localhost % hdfsPort).str();
   auto readFailErrorMessage =
-      (boost::format(
-           "Unable to open file %s. got error: HdfsIOException: InputStreamImpl: cannot open file: %s.\t"
-           "Caused by: Hdfs::HdfsRpcException: HdfsFailoverException: Failed to invoke RPC call \"getBlockLocations\" on server \"%s\"\t\t"
-           "Caused by: HdfsNetworkConnectException: Connect to \"%s\" failed") %
-       destinationPath % destinationPath % serverAddress % serverAddress)
-          .str();
-  auto builderErrorMessage =
-      (boost::format(
-           "Unable to connect to HDFS: %s, got error: Hdfs::HdfsRpcException: HdfsFailoverException: "
-           "Failed to invoke RPC call \"getFsStats\" on server \"%s\"\tCaused by: "
-           "HdfsNetworkConnectException: Connect to \"%s\" failed") %
-       serverAddress % serverAddress % serverAddress)
+      (boost::format("Unable to open file %s. got error: EOFException:") %
```

Comment:
It is hard to check detailed error message, because the error message is changed in different method call.